src/App.js

```diff
@@ -1,8 +1,58 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import "./App.css";
 
+import Button from './components/Button'
+import UserList from './components/UserList'
+import UserCard from "./components/UserCard";
+
 function App() {
-  return <div>Good luck dear students!</div>;
+  const [usersNames, setUsersNames] = useState([])
+  const [users, setUsers] = useState({})
+  const [currentUser, setCurrentUser] = useState({})
+  const [isLoading, setIsLoading] = useState(true)
+
+  useEffect(() => {
+    handleUsers()
+  }, [])
+
+  const handleUsers = async() => {
```

Comment:
Just a renaming suggestion
```suggestion
  const getUsers = async() => {
```